bin/postgres-ha/pre-bootstrap.sh

```diff
@@ -204,7 +204,7 @@ set_pg_user_credentials() {
         env_check_err "PGHA_USER_PASSWORD"
     fi
 
-    if [[ -d "/pgconf/pguser" ]]
+    if [[ -d "/pgconf/pgsuper" ]]
```

Comment:
I would strongly advise that we spell out `superuser` -- let's make it so someone reviewing it has to think less.